ansible/roles/cli/tasks/download_cli_mac.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: "ensure cli sub directory structure exists"
+  file:
+    path: "{{ cli_nginx_dir }}/mac/amd64"
+    state: directory
+
+- name: "download cli into ngnix"
+  local_action: >
+    get_url
+    url="https://github.com/openwhisk/openwhisk-cli/releases/download/latest/OpenWhisk_CLI-latest-darwin-amd64.zip"
+    dest="{{ cli_nginx_dir }}/mac/amd64/OpenWhisk_CLI-mac.zip"
+    validate_certs=False
```

Comment:
It was not done before and hence following the same workflow. We can validate them if needed since now CLI is part of separate repo. Thoughts?